build.boot

```diff
@@ -4,6 +4,8 @@
 (def +node-version+ (or (System/getenv "BUILD_NODE_VERSION")
                         "11.13.0"))
 
+(def +build-static-binary+ (System/getenv "BUILD_STATIC_BINARY"))
```

Comment:
let's do `(boolean (System/getenv "BUILD_STATIC_BINARY"))`

Reply:
I had something like that but then it needs to be converted to a string for `dosh` (it is asserting on it) so I went back to this and handle things in `package.js`

Reply:
then let's do `(-> (System/getenv "BUILD_STATIC_BINARY") boolean str)`.

I really don't want random strings leaking around.

Reply:
sounds good